src/test/python/testpropertyvalue.py

```diff
@@ -1,6 +1,8 @@
 import sys
 sys.path.append("./generated")
 
+from uuid import UUID
```

Comment:
If you don't import UUID, can you still import winrt.windows.foundation successfully? Or would it only fail if you use create_guid or other UUID APIs without importing it?

Reply:
The convert function will import the `uuid` module if needed so users don't need to import it.